osgp/protocol-adapter-dlms/osgp-protocol-simulator-dlms/simulator/dlms-device-simulator/src/main/java/org/opensmartgridplatform/simulator/protocol/dlms/cosem/MBusEventLog.java

```diff
@@ -129,10 +129,10 @@ private void initBufferData() {
     this.bufferData.add(Arrays.asList(this.getNextDateTime(), (short) 255));
 
     // Add all events.
-    this.addEvents((short) 100, (short) 106);
```

Comment:
Moeten deze wel verwijderd worden, want de permanent errors bestaan nog met 106,116,126 en 136?

Reply:
They are not removed. The range is expanded with 1

Reply:
So thats how it works. Then the changes are approved.